lib/android/app/src/main/java/com/reactnativenavigation/views/stack/topbar/titlebar/TitleComponentMeasurer.kt

```diff
@@ -0,0 +1,32 @@
+package com.reactnativenavigation.views.stack.topbar.titlebar
+
+import kotlin.math.max
+
+fun resolveTitleBoundsLimit(
+        defaultStartMargin: Int,
```

Comment:
Seems like we're always calling this function with `defaultMargin` which is a **private** constant - no need to pass it as a param to the function.

You can pull it out of the class scope and make the conversion without the use of a context like it's done [here](https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/main/java/com/reactnativenavigation/options/params/DensityPixel.java#L10)